### DISCUSSION TITLE
When to expect Support vor Vue 3 & Nuxt 3 ? Workarounds for nuxt modules?

### DISCUSSION THREAD

**Discussion by awacode21** [cypress-io/cypress#30020] [Questions and Help] [answered]
Honestly Vue 3 and also Nuxt 3 are out there for quite a while now. Also Vue 2 has reached EOL and is no longer actively maintained.

But Cypress is still only supporting Vue 2 and vue 2 with nuxt 2.

When do you think we can expect the update in support here?

I am currently in struggle testing my nuxt 3 components with cypress on my customers project. They set it up to run "somehow". But now we use @nuxt/icon component and this is where it all crashed as the component obviously cannot be found in cypress.

Are there any known workarounds to setup cypress for nuxt 3 with nuxt modules/components?

**Comment by MikeMcC399** [comment]
@awacode21 

- This is tracked in https://github.com/cypress-io/cypress/issues/23619, however I don't see any progress there unfortunately.

**Comment by jennifer-shehane** [comment, marked as answer]
There’s a section in the docs explaining how to get this to work to text Nuxt 3, open an issue if this doesn’t work for you. https://docs.cypress.io/app/component-testing/vue/overview#Using-Cypress-with-Nuxt
